src/main/java/org/jenkinsci/plugins/oic/OicSecurityRealm.java

```diff
@@ -320,8 +320,11 @@ ClientAuthenticationMethod toClientAuthenticationMethod() {
      */
     private transient ProxyAwareResourceRetriever proxyAwareResourceRetriever;
 
-    private List<OicQueryParameterConfiguration> loginQueryParamNameValuePairs;
-    private List<OicQueryParameterConfiguration> logoutQueryParamNameValuePairs;
+    @SuppressFBWarnings(value = "SE_BAD_FIELD", justification = "we are not using standard serialization")
```

Comment:
OicSecurityRealm is marked as `Serializable` and then the `LoginQueryParameters` fail as their parent class does not have a zero arg constructor...  but we do not deserialize like this with xstream and these classes are not sent via Remoting so there is no need for this class to be Serializable...